stage0/run.go

```diff
@@ -47,21 +43,17 @@ import (
 	"github.com/coreos/rocket/pkg/lock"
 	ptar "github.com/coreos/rocket/pkg/tar"
 	"github.com/coreos/rocket/version"
-
-	"github.com/coreos/rocket/stage0/stage1_init"
-	"github.com/coreos/rocket/stage0/stage1_rootfs"
 )
 
 const (
-	initPath  = "stage1/init"
+	initPath  = "stage1/rootfs/init"
```

Comment:
Because of https://github.com/appc/spec/issues/154 I'm going back to using annotations.

Also, annotations are per-image, eventHandlers are per-app.  An stage1 .aci doesn't really contain an app, it's awkward creating an app object having just eventHandlers, but not so awkward to use annotations and omit an app object altogether in the stage1 aci manifest.